### PR TITLE
Chore: map enums to union string in generated code

### PR DESCRIPTION
**Pull request by nguyentoanit**

### CODE REVIEW

**Comment by moltar** [comment]
Task linked: [GHORG-52 GH Issue #60: export all enums](https://app.clickup.com/t/2250003/GHORG-52)

File: src/api-models/fba-inventory-api-model/base.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /* eslint-disable */
 /**
  * Selling Partner API for FBA Inventory
- * The Selling Partner API for FBA Inventory lets you programmatically retrieve information about inventory in Amazon\'s fulfillment network.
+ * The Selling Partner API for FBA Inventory lets you programmatically retrieve information about inventory in Amazon\'s fulfillment network. Today this API is available only in the North America region. In 2021 we plan to release this API in the Europe and Far East regions.
```
Comment (moltar) [inline src/api-models/fba-inventory-api-model/base.ts:5]: Interesting that there are other changes here, unrelated to ENUM and unions. That means our auto updater is not working?

**Review by moltar — CHANGES_REQUESTED** [review]
This looks good. Good job figuring this out.

The only thing is - please isolate the changes. There are other unrelated changes here, related to updated models and even new models (tokens). Only include the changes related to enum and unions.

Then we can have a separate PR to add the other updates.

And then figure out why our auto-updates aren't working.

File: src/api-models/fba-inventory-api-model/base.ts (hunk 1 above)
Comment (nguyentoanit) [reply src/api-models/fba-inventory-api-model/base.ts]: I've removed unrelated changes. Thanks.

Comment (nguyentoanit) [reply src/api-models/fba-inventory-api-model/base.ts]: > That means our auto updater is not working?

I re-generated models. Therefore, there were unrelated changes on this PR.

**Review by moltar — APPROVED** [review]

**Pull request merged**